deepmd/utils/data.py

```diff
@@ -621,43 +633,49 @@ def _load_data(
         if path.is_file():
             data = path.load_numpy().astype(dtype)
             try:  # YWolfeee: deal with data shape error
-                if atomic:
-                    if type_sel is not None:
-                        # check the data shape is nsel or natoms
-                        if data.size == nframes * natoms_sel * ndof_:
-                            if output_natoms_for_type_sel:
-                                tmp = np.zeros(
-                                    [nframes, natoms, ndof_], dtype=data.dtype
-                                )
-                                sel_mask = np.isin(self.atom_type, type_sel)
-                                tmp[:, sel_mask] = data.reshape(
-                                    [nframes, natoms_sel, ndof_]
-                                )
-                                data = tmp
-                            else:
-                                natoms = natoms_sel
-                                idx_map = idx_map_sel
-                                ndof = ndof_ * natoms
-                        elif data.size == nframes * natoms * ndof_:
-                            if output_natoms_for_type_sel:
-                                pass
+                if key == "coord" and self.multistru:
```

Comment:
Do we assume aparam and other properties are the same for all conformations?

Reply:
I haven't thought of a situation where there are different other properties(aparam fparam) of all conformations. The reason I add this feature is that when one only have molecules in SMILES format, one may generate several possible conformations using tools such as rdkit. Do you have more better ideas?